.travis.yml

```diff
@@ -80,3 +72,4 @@ cache:
   directories:
     - $HOME/.pyenv
     - $HOME/.cache/pip
+    - $HOME/build/mitmproxy/.tox
```

Comment:
I'm not sure we should cache .tox; having a clean environment every time will help to catch issues we might not otherwise notice.  99% of what's installed are wheels anyway (already cached by pip), with cryptography being one of the few notable exceptions.  Also, if we change OpenSSL versions, we'd have to dump the caches, otherwise cryptography won't get rebuilt, and then things get fun.